test/test_qml.cpp

```diff
@@ -157,11 +179,35 @@ class Setup : public QObject
       qmlInit( engine );
       engine->rootContext()->setContextProperty( QStringLiteral( "dataDir" ), mDataDir );
 
+
+      qmlRegisterType<QgsSnappingUtils>( "org.qgis", 1, 0, "SnappingUtils" );
+      qmlRegisterType<QgsMapLayerProxyModel>( "org.qgis", 1, 0, "MapLayerModel" );
+      qmlRegisterType<QgsVectorLayer>( "org.qgis", 1, 0, "VectorLayer" );
+      qmlRegisterType<QgsMapThemeCollection>( "org.qgis", 1, 0, "MapThemeCollection" );
+      // qmlRegisterType<QgsLocatorProxyModel>( "org.qgis", 1, 0, "QgsLocatorProxyModel" );
```

Comment:
Remove?

Reply:
I', not sure how to register this one ?! its registered in `qgismobileapp.cpp` but couldn't find its `#include` signature.

Reply:
@mohsenD98 , you can use QGIS' excellent API documentation site to get the answer: https://api.qgis.org/api/classQgsLocatorProxyModel.html

At the top, you'll see the include, in this case: `#include <qgslocatormodel.h>`

Reply:
thank you, fantastic :+1: